cloudfoundry-operations/src/test/resources/fixtures/base-manifest.yml

```diff
@@ -0,0 +1,9 @@
+---
+buildpack: https://github.com/cloudfoundry/java-buildpack#v4.5.2
+memory: 1G
+path: /target/demo.jar
+env:
+  JBP_CONFIG_OPEN_JDK_JRE: '{ jre: { version: 1.8.0_+ } }'
+
+#applications:
```

Comment:
If you uncomment these lines, the test will pass, but these lines shouldn't be needed, I think.